.github/workflows/int-and-connected-test-run.yml

```diff
@@ -266,7 +272,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
         with:
-          status: ${{ job.status }}
+          status: ${{ (needs.test_check.outputs.job-status == 'success' && needs.test_connected.outputs.job-status == 'success' && needs.deploy_test_integration.outputs.job-status == 'success') && 'success' || 'failure' }}
```

Comment:
I'd like to understand more about why this is needed.  We can do that outside of this PR.

Reply:
This looks like the same thing we ended up doing for catalog. The problem (as I recall it) is that `job.status` is the status of the previous job, not of all the `needs` for this job. If you have an `always()` job, then you don't know if any `needs` failed or not, since it runs no matter what. The code we did for this is this PR https://github.com/DataBiosphere/terra-data-catalog/pull/107